apps/site/next.locales.mjs

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
IMO this file should be on package i18n. So it's can be share between multiple apps.

Reply:
agree